CONTRIBUTING.md

```diff
@@ -0,0 +1,16 @@
+# Contributing to Substrait
+
+Welcome!
+
+## Dependencies
+
+There's no formal set of dependencies for substrait, but here are some that are useful to have:
+
+* [`buf`](https://docs.buf.build/installation) for easy generation of proto serialization/deserialization code
+* [`protoc`](https://grpc.io/docs/protoc-installation/), used by `buf` and usable independent of `buf` A Python environment with [the website's `requirements.txt`](https://github.com/substrait-io/substrait/blob/main/site/requirements.txt) dependencies installed if you want to see changes to the website locally
```

Comment:
Looks like there's a newline/bullet point missing before "A Python environment".

Reply:
I mean, shouldn't it be:

```
 * [`protoc`](https://grpc.io/docs/protoc-installation/), used by `buf` and usable independent of `buf`
 * A Python environment with [the website's `requirements.txt`](https://github.com/substrait-io/substrait/blob/main/site/requirements.txt) dependencies installed if you want to see changes to the website locally 
```

i.e.

 * [`protoc`](https://grpc.io/docs/protoc-installation/), used by `buf` and usable independent of `buf`
 * A Python environment with [the website's `requirements.txt`](https://github.com/substrait-io/substrait/blob/main/site/requirements.txt) dependencies installed if you want to see changes to the website locally 

rather than

* [`protoc`](https://grpc.io/docs/protoc-installation/), used by `buf` and usable independent of `buf` A Python environment with [the website's `requirements.txt`](https://github.com/substrait-io/substrait/blob/main/site/requirements.txt) dependencies installed if you want to see changes to the website locally

Reply:
Oh, gross, yeah. Not sure how I missed that!

Reply:
Too many `gw` keystrokes.

Reply:
Fixed.